compiler/rustc_error_messages/locales/en-US/parse.ftl

```diff
@@ -238,6 +238,7 @@ parse_const_let_mutually_exclusive = `const` and `let` are mutually exclusive
 
 parse_invalid_expression_in_let_else = a `{$operator}` expression cannot be directly assigned in `let...else`
 parse_invalid_curly_in_let_else = right curly brace `{"}"}` before `else` in a `let...else` statement not allowed
+parse_extra_if_in_let_else = remove the `if` to construct a `let...else` statement
```

Comment:
```suggestion
parse_extra_if_in_let_else = remove the `if` if you meant to write a `let ... else` statement
```

Reply:
Maybe this wording is a bit better? Idk

Reply:
Re: the space between let and else, the line above it already doesn't include them. Not sure if you want me to stay consistent?

Reply:
Re: wording, I searched around, looks like that's more consistent with what else is in here, thanks.

Reply:
uhhh i guess just keep it as-is, just apply the wording changes if you think it's clearer

Reply:
Yeah, I keep forgetting to rebless the tests. There'll be one more force-push